ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-periodics.yaml

```diff
@@ -270,6 +270,57 @@ periodics:
           sources:
           - secret:
               name: cluster-secrets-azure
+  - agent: kubernetes
+    cron: ""
+    decorate: true
+    interval: 24h
+    name: azure-build-node-image-rhel-310-not-running-under-test
```

Comment:
I expected we would merely run tests in openshift-azure w/o specifying RUNNING_UNDER_TEST. Nothing to do with the node build job. I would prefer we continue building images tested w/o RUNNING_UNDER_TEST.